addons/website_event_questions/report/report_event_question_view.xml

```diff
@@ -40,6 +40,9 @@
                         <filter string="Event" name="group_by_event" context="{'group_by': 'event_id'}" />
                         <filter string="Question" name="group_by_question" context="{'group_by': 'question_id'}"/>
                         <filter string="Answer" name="group_by_answer" context="{'group_by': 'answer_id'}" />
+                        <filter name="exclude_cancelled"
+                                string="Exclude cancelled registrations"
+                                domain="[['attendee_id.state', '!=', 'cancel']]" />
```

Comment:
The domain is usually written as a list of tuple and not a list of list (even if both likely work since elements accessed by index).

Other than that, I don't know anything about this module and I don't know much about search views either, so it looks good to me unless you have a specific question or concern?